src/core_init_atmosphere/mpas_init_atm_cases.F

```diff
@@ -1857,7 +1834,7 @@ subroutine init_atm_case_mtn_wave(dminfo, mesh, nCells, nVertLevels, state, diag
       real (kind=RKIND), dimension(nVertLevels ) :: zu, dzw, rdzwp, rdzwm
 
       real (kind=RKIND) :: d1, d2, d3, cof1, cof2
-      real (kind=RKIND) :: um, us,  rcp, rcv
+      real (kind=RKIND) :: um, vm, us, vs, rcp, rcv
```

Comment:
It looks like we can safely remove the `us` and `vs` variables.

Reply:
us and vs removed.